Source/Parallelization/WarpXComm.cpp

```diff
@@ -79,7 +79,7 @@ WarpX::UpdateAuxilaryData ()
             MultiFab::Subtract(dBz, *Bfield_cp[lev][2], 0, 0, 1, ng);
 
             const Real* dx = Geom(lev-1).CellSize();
-            const int ref_ratio = refRatio(lev-1)[0];
+            const int rr = refRatio(lev-1)[0];
```

Comment:
Do you think we should use a name more explicit, e.g., `refinement_ratio` or something like that?

Reply:
I changed this.